### PR TITLE
Create Node5 Image

### PR DESCRIPTION
**Pull request by ksafranski**
@TomFrost

### CODE REVIEW

File: 5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN apk add git openssh-client
+
+# native build deps
+RUN apk add make gcc
+
+# scripting tools
+RUN apk add bash
+
+# update NPM
+RUN npm install -g npm@3.0-latest
```
Comment (TomFrost) [inline 5/Dockerfile]: Is there a 3-latest we can use, so we get minor version updates? Not sure what they publish. If not, disregard :)

**Comment by TomFrost** [comment]
:accept: if that's the appropriate way to get the best npm 3 release!

File: 5/Dockerfile (hunk 1 above)
Comment (ksafranski) [inline 5/Dockerfile]: only saw `npm@3.0-next` and `npm@3.0-latest`...

Comment (TomFrost) [inline 5/Dockerfile]: Cool, that's probably just how they publish minor versions then.

**Pull request merged**